FinanceTracker/FinanceTracker/Models/Account.cs

```diff
@@ -34,6 +34,10 @@ public void UpdateBalance(Operation operation)
             {
                 Balance -= operation.Amount;
             }
+
+            Operations.Add(operation);
         }
+
+        public List<Operation> GetOperations() { return Operations; }
```

Comment:
You can make Operations as property with public get and private set, and you wont need this method at all